cloud/docker/docker.py

```diff
@@ -772,6 +778,7 @@ def get_start_params(self):
             'privileged': self.module.params.get('privileged'),
             'links': self.links,
             'network_mode': self.module.params.get('net'),
+            'read_only': self.module.params.get('read_only'),
```

Comment:
I feel like this should be gated behind an `ensure_capability` check like some of the other HostConfig parameters added [further down](https://github.com/ansible/ansible-modules-core/blob/devel/cloud/docker/docker.py#L782-L784), because it was added in a recent revision of the Docker API (and docker-py API).

At a glance it looks like `read_only` was added to the Docker API in [1.17](https://docs.docker.com/reference/api/docker_remote_api/#v1-17). I'm checking for the docker-py version presently.